.github/workflows/spark.yaml

```diff
@@ -0,0 +1,28 @@
+name: spark
```

Comment:
naming: "spark" is too general, I would call this "spark metadata client checks"

Reply:
not necessarily in this field, I just want it to be easier to understand where it appears on github:
![image](https://user-images.githubusercontent.com/3149885/122176746-9cb8cf00-ce8d-11eb-9a9d-981a134e53e7.png)